app/src/main/java/rick_and_morty/data/api/ApiConstants.kt

```diff
@@ -0,0 +1,6 @@
+package rick_and_morty.data.api
+
+object ApiConstants {
+    const val RICKANDMORTY_URL = "https://rickandmortyapi.com/api/"
+    const val CHARACTERS_ENDPOINT = "character"
```

Comment:
I would hardcode both of these as they are not reused.

Reply:
move to retrofit single file

Reply:
Done